common/lib/common-utils/tsconfig.esnext.json

```diff
@@ -2,6 +2,7 @@
     "extends": "./tsconfig.json",
     "compilerOptions": {
         "outDir": "./lib",
-        "module": "esnext"
+        "module": "esnext",
+        "types": [ "node" ]
```

Comment:
node types are intentionally omitted here to make it harder to add dependencies.  may need to add some declarations to the _node files as a workaround

Reply:
Alright I added a typings file and an IsomorphicPerformance type to try to cover this. Is this okay?